nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/model/PublicationDetails.java

```diff
@@ -85,18 +85,11 @@ public static PublicationDetails from(CandidateDao candidateDao) {
         builder()
             .withId(dbCandidate.publicationId())
             .withPublicationBucketUri(dbCandidate.publicationBucketUri())
-            .withTitle(dbDetails.title())
-            .withStatus(dbDetails.status())
-            .withLanguage(dbDetails.language())
-            .withAbstract(dbDetails.abstractText())
             .withPublicationDate(PublicationDate.from(dbCandidate.getPublicationDate()))
-            .withPageCount(PageCount.from(dbDetails.pages()))
             .withIsApplicable(dbCandidate.applicable())
             .withPublicationChannel(PublicationChannel.from(candidateDao))
             .withNviCreators(nviCreators)
-            .withCreatorCount(dbDetails.contributorCount())
-            .withTopLevelOrganizations(topLevelOrganizations)
-            .withModifiedDate(dbDetails.modifiedDate());
+            .withTopLevelOrganizations(topLevelOrganizations);
 
     if (nonNull(dbDetails)) {
       return getPublicationDetailsWithMigratedFields(dbDetails, builder);
```

Comment:
The "removed" fields are set here if the record is migrated, otherwise we do not have them anyway.